test/extended/security/sysctl.go

```diff
@@ -34,4 +35,99 @@ var _ = g.Describe("[sig-arch] [Conformance] sysctl", func() {
 		t.Entry("net.ipv4.ping_group_range", "net.ipv4.ping_group_range", "1\t0", "/proc/sys/net/ipv4/ping_group_range"),
 		t.Entry("net.ipv4.ip_unprivileged_port_start", "net.ipv4.ip_unprivileged_port_start", "1002", "/proc/sys/net/ipv4/ip_unprivileged_port_start"),
 	)
+
+	t.DescribeTable("pod with sysctl should not affect node sysctl", func(sysctl, value, path string) {
+		f := oc.KubeFramework()
+
+		nodePod := frameworkpod.CreateExecPodOrFail(f.ClientSet, f.Namespace.Name, "sysctl-pod-", func(pod *v1.Pod) {
+			pod.Spec.Volumes = []v1.Volume{
+				{Name: "sysvolume", VolumeSource: v1.VolumeSource{HostPath: &v1.HostPathVolumeSource{Path: "/proc"}}},
+			}
+			pod.Spec.Containers[0].VolumeMounts = []v1.VolumeMount{{Name: "sysvolume", MountPath: "/host/proc"}}
+		})
+
+		nodeOutputBeforeSysctlAplied, err := oc.AsAdmin().Run("exec").Args(nodePod.Name, "--", "/bin/bash", "-c", "cat /host/"+path).Output()
```

Comment:
```suggestion
		nodeOutputBeforeSysctlAplied, err := oc.AsAdmin().Run("exec").Args(nodePod.Name, "--", "cat", "/host/"+path).Output()
```
? (and likewise elsewhere)